packages/create-wagmi/templates/vite-react/_dot_npmrc

```diff
@@ -0,0 +1 @@
+strict-peer-dependencies = false
```

Comment:
what does this do in this case? prevent logging to peer dep warnings to the console on install?

Reply:
By default, npm doesn't allow incompatible peer dependencies.